### PR TITLE
fix: Adds json schema validator for `timezone` property

### PR DESCRIPTION
**Pull request by KillDozerX2**
Fork of #17, @nyoungstudios commits are here as well.

### CODE REVIEW

**Comment by KillDozerX2** [comment]
@pgrzesik if this could be merged, I can continue with my fork to finish #13 and #21 as well.

File: index.js [hunk 1]
```diff
@@ -121,6 +122,17 @@ function convertCrontabs() {
 class ServerlessLocalCrontabs {
   constructor(serverless, options) {
     this.serverless = serverless;
+    if (this.serverless.configSchemaHandler) {
```
Comment (pgrzesik) [inline index.js]: Sorry for not mentioning it at first - I've just double checked and there can be a situation where the `configSchemaHandler` is defined, but the `defineFunctionEventProperties` method is not - let's also check for the existence of this method before trying to call it, okay? It was added in `2.11.0` so let's not break users that are on lower versions: https://github.com/serverless/serverless/blob/main/CHANGELOG.md#2110-2020-11-06

**Review by pgrzesik — CHANGES_REQUESTED** [review]
thanks @KillDozerX2 - I have only one minor comment, please let me know what do you think

File: index.js (hunk 1 above)
Comment (KillDozerX2) [reply index.js]: @pgrzesik done

**Review by pgrzesik — APPROVED** [review]
thanks @KillDozerX2 - looks good 👍

**Pull request merged**